### PR TITLE
Capacitor

### PR DESCRIPTION
**Pull request by dillchen**
supersedes #2206

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/quill/render_quill_delta.ts [hunk 1]
```diff
@@ -106,7 +108,12 @@ export const renderQuillDelta = (
                       target: openLinksInNewTab ? '_blank' : '',
                       noreferrer: 'noreferrer',
                       noopener: 'noopener',
-                      onclick: (e) => {
+                      onclick: async (e) => {
+                        console.log('click');
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/views/components/quill/render_quill_delta.ts:112]: `console.log('reviewed for the uber eats');`

Comment (CowMuon) [reply packages/commonwealth/client/scripts/views/components/quill/render_quill_delta.ts:112]: I believe that is a comment, not a review.

File: packages/commonwealth/server/scripts/setupServer.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ const setupServer = (
     if (typeof addr === 'string') {
       log.info(`Listening on ${addr}`);
     } else {
-      log.info(`Listening on port ${addr.port}`);
+      log.info(`Listening on port ${addr?.port}`);
```
Comment (CowMuon) [inline packages/commonwealth/server/scripts/setupServer.ts:47]: nice catch here.

**Review by CowMuon — APPROVED** [review]
Overall this should be fine. As previously discussed with everyone (Dillon, Ray, Zak, Jake) this largely doesn't impact current development, but is needed for the road ahead.  The main reason we didn't pull this in before was React was priority, and I wanted to make sure we are all aligned on what approach to mobile will be. LGTM on 2/17. 

Approved contingent on minor changes as noted.

**Comment by dillchen** [comment]
This should be ready!

**Comment by CowMuon** [comment]
I am still seeing old chat server in webpack.config.mobile.js? Is there a commit that didn't get pushed up for that?

**Comment by dillchen** [comment]
> I am still seeing old chat server in webpack.config.mobile.js? Is there a commit that didn't get pushed up for that?

Missed this, I think we only discussed this verbally so did not address

**Comment by dillchen** [comment]
Wowowowow

**Pull request merged**